examples/quickstart/notebooks/quickstart.ipynb

```diff
@@ -574,7 +603,43 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "After the training pipeline run has finished, the trained model will have been deployed using MLflow and is ready for prediction requests. We can use `zenml model-deployer models list` to get an overview of all currently deployed models:"
+    "And now let's replace the SVC trainer with the Tree trainer."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "training_pipeline(\n",
+    "    training_data_loader=training_data_loader(),\n",
+    "    trainer=tree_trainer_mlflow(),\n",
+    "    evaluator=evaluator(),\n",
+    "    deployment_trigger=deployment_trigger(),\n",
+    "    model_deployer=mlflow_model_deployer_step(),\n",
+    ").run()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Notice that the second pipeline ran slightly faster than the first? That's because ZenML understands that the first few steps of your pipeline are unchanged, so it just reloads them from your previous run and goes straight to the trainer part. This saves valuable time as you iterate your pipeline"
```

Comment:
```suggestion
    "Notice that the second pipeline ran slightly faster than the first? That's because ZenML understands that the first few steps of your pipeline are unchanged, so it just reloads them from your previous run and goes straight to the trainer part. This saves valuable time as you iterate on your pipeline."
```